api/v1beta3/ratelimitpolicy_types.go

```diff
@@ -32,8 +32,8 @@ import (
 const (
 	EqualOperator      WhenConditionOperator = "eq"
 	NotEqualOperator   WhenConditionOperator = "neq"
-	StartsWithOperator WhenConditionOperator = "startswith"
-	EndsWithOperator   WhenConditionOperator = "endswith"
+	StartsWithOperator WhenConditionOperator = "startsWith"
+	EndsWithOperator   WhenConditionOperator = "endsWith"
```

Comment:
I'm not 100% about changing these - they're not used elsewhere but it's a little confusing using some within CEL predicates and others for `PatternExpression`s

Reply:
Unfortunately we can't remove the `WhenCondition` because we import the `AuthConfig` which still has selector expressions present.. so I've left it so it can support both.. but if we're not using these types I could remove them

Reply:
Unless if a user sets them we just ignore them?

Reply:
Up to you. I still believe that the operator can generate CEL expressions out of `namePatterns` instead of `WhenConditions` and we can :fire:  `WhenConditions`. 

But can be done later on.